statsdaemon.go

```diff
@@ -471,7 +472,7 @@ func parseMessage(data []byte) []*Packet {
 		}
 
 		packet := &Packet{
-			Bucket:   sanitizeBucket(*prefix + string(name)),
+			Bucket:   sanitizeBucket(*prefix + string(name) + *postfix),
```

Comment:
Good catch @omega. Not sure how I missed that in debugging, submitted #65 to fix.